packages/hosted-form-v2/src/payment/Instrument.ts

```diff
@@ -3,6 +3,32 @@ export enum InstrumentType {
     ManualPayment = 'manual_payment',
 }
 
+export enum OfflinePaymentMethodId {
```

Comment:
We might ned to rename this as https://github.com/bigcommerce/checkout-sdk-js/pull/2743/files#diff-b4ae89fa8bc508c57bcd08f924ebef6e94de9ee5e67e31def25515f3ed47f3c8R3 here it can get a bit confusing to read as paymentMethodId implies it's an id but here its an object.

Reply:
Renamed it to the plural form, and update the helper function to use `in` method.
```
export const isOfflinePaymentMethodId = (id: string): id is OfflinePaymentMethodIds => {
    return id in OfflinePaymentMethodIds;
};
```